completion/available/go.completion.bash

```diff
@@ -1,281 +1,13 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
```

Comment:
Surely we want to keep `#!/usr/bin/env bash` ?

Reply:
This was one I was not _quite_ sure of, happy to hear people's thoughts. I saw some inconsistency around which is used and I see you opened issue #1491 - I'll put some of my thoughts there